content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -192,6 +200,12 @@ characters.
 
 A Job also needs a [`.spec` section](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
 
+### Job Labels
+
+In releases prior to 1.27, Jobs used unprefixed labels ie `job-name` and `controller-uid`. 
+In 1.27, Job labels will have `batch.kubernetes.io` prefix for `job-name` and `controller-uid`.
```

Comment:
Probably should start with this line

Reply:
Not sure what you are asking here.

Reply:
The first line of the documentation is what is the current behavior (the behavior for 1.27).
But maybe sig-docs has a different guidance.

Reply:
I see.  Daylight savings has not been good on my brain.

Reply:
the prefix is `batch.kubernetes.io/` to be precise :wink:

Reply:
Done.